modules/ROOT/pages/design-create-publish-api-raml-editor.adoc

```diff
@@ -9,54 +12,63 @@ Within Anypoint Platform, your user ID must be assigned the _Design Center Devel
-. Choose either the RAML or OAS specification. 
+. If you are creating a REST API specification:
+.. Select the *I'm comfortable designing it on my own* option.
+.. Choose either the RAML or OAS specification. 
 +
 API Designer creates the stubs of your API specification using the specification type and version you choose.
 +
 By default API Designer creates a RAML 1.0 file. You can change the file to RAML 0.8 or OAS after the project is created. Projects for creating API specifications can contain a mix of RAML, OAS, and other types of files.
++
+. If you are creating an AsyncAPI specification:
++
+* Select either YAML or JSON as the project language.
+
+The text editor opens. 
 
-*Result:* The text editor opens. The editor is divided into three panels:
+The API Designer text editor is divided into panels:
 
-* The left panel lists the files in your project.
+* The left panel lists the files and dependencies in your project.
 +
-The stub specification file is listed and set as the root file of the project. You can change the name (but not the extension) of this file by right-clicking the three dots to the right of the name and selecting *Rename*.
+The stub specification file is listed and set as the root file of the project. You can change the name (but not the extension) of this file by clicking the overflow menu icon (image:manage-dependency-versions-design-center-018a5.png[width=0.75%,height=0.75%,title="Screenshot of the overflow menu icon"]) beside the specification name and selecting *Rename*.
 +
 The left panel also lists the file `exchange.json`, which is a file that contains metadata required by Anypoint Exchange at the time that you publish your project to Exchange. This file is read-only.
 +
 [TIP]
 ====
 
-You can create more than one API specification or related file in a single project. Click the plus sign in the Files panel to create a new file or click the dots that are next to the plus sign to import a file.
+You can create more than one API specification or related file in a single project. Click the plus sign in the Files panel to create a new file.
 
 If you do create more than one specification file, you must specify one of them as the root file of your project.
 
 ====
-
-* The central panel displays the editor in which you create the API specification. Appearing in the editor when you create a project or start a new API specification are the first two lines of a specification file in RAML or OAS, depending on which you selected when you created the project. You can change these lines (as well as the file extension) if you want to switch from RAML to OAS or vice-versa.
-* The right panel lists the types and resources that are in the API specification that is in the central panel.
++
+For more information about adding Exchange dependencies, see "Adding Dependencies to an API-Specification Project", which is linked to from the *See also* section at the end of this topic.
```

Comment:
I would just put the link here and not say "which is linked to from the *See also* section at the end of this topic." Consider removing the link from the See Also section, depending on whether it is needed for other contexts in the topic.

Reply:
Yes, they've done this throughout the doc. I think we can change this later.